std/assembly/util/string.ts

```diff
@@ -69,20 +70,17 @@ export const enum CharCode {
   z = 0x7A
 }
 
-// 9 * 8 = 72 bytes
+// 23 * 8 = 184 bytes
 // @ts-ignore: decorator
-@lazy
-const Powers10Hi: f64[] = [1, 1e32, 1e64, 1e96, 1e128, 1e160, 1e192, 1e224, 1e256, 1e288];
-// 32 * 8 = 256 bytes
-// @ts-ignore: decorator
-@lazy
-const Powers10Lo: f64[] = [
+@lazy const Powers10: f64[] = [
   1e00, 1e01, 1e02, 1e03, 1e04, 1e05, 1e06, 1e07, 1e08, 1e09,
   1e10, 1e11, 1e12, 1e13, 1e14, 1e15, 1e16, 1e17, 1e18, 1e19,
-  1e20, 1e21, 1e22, 1e23, 1e24, 1e25, 1e26, 1e27, 1e28, 1e29,
-  1e30, 1e31
+  1e20, 1e21, 1e22,
 ];
 
+// @ts-ignore
+@lazy const powers10 = Powers10.dataStart as usize;
```

Comment:
In other code, like in the log PR that's currently open, we try to avoid emitting unnecessary globals like this one. Even though it's `const` here, since `.dataStart` isn't, this will become one.